Runtime/gestures/mouse_tracking.cs

```diff
@@ -4,23 +4,43 @@
 using Unity.UIWidgets.ui;
 
 namespace Unity.UIWidgets.gestures {
-    
     public delegate void PointerHoverEventListener(PointerHoverEvent evt);
 
     public delegate void PointerEnterEventListener(PointerEnterEvent evt);
 
     public delegate void PointerExitEventListener(PointerExitEvent evt);
-    
-    
+
+#if UNITY_EDITOR
+    public delegate void PointerDragFromEditorEnterEventListener(PointerDragFromEditorEnterEvent evt);
+
+    public delegate void PointerDragFromEditorHoverEventListener(PointerDragFromEditorHoverEvent evt);
+
+    public delegate void PointerDragFromEditorExitEventListener(PointerDragFromEditorExitEvent evt);
+
+    public delegate void PointerDragFromEditorReleaseEventListener(PointerDragFromEditorReleaseEvent evt);
+#endif
+
+    /// The annotation object used to annotate layers that are interested in mouse
+    /// movements.
+    /// This is added to a layer and managed by the [Listener] widget.
     public class MouseTrackerAnnotation {
         public MouseTrackerAnnotation(
             PointerEnterEventListener onEnter = null,
             PointerHoverEventListener onHover = null,
-            PointerExitEventListener onExit = null
+            PointerExitEventListener onExit = null,
+            PointerDragFromEditorEnterEventListener onDragFromEditorEnter = null,
```

Comment:
there are potential compile errors on other build target platforms. Solve all of them first